docs/2-building-blocks-of-oop-part-1/hometask/src/Triangle.ts

```diff
@@ -11,17 +11,15 @@ export class Triangle extends Shape {
     const side1 = this.points[0].distance(this.points[1]);
     const side2 = this.points[1].distance(this.points[2]);
     const side3 = this.points[2].distance(this.points[0]);
-    if (Math.round(side1) === Math.round(side2) && Math.round(side2) === Math.round(side3)) {
+    if (side1.toFixed(1) === side2.toFixed(1) && side2.toFixed(1) === side3.toFixed(1)) {
       return "equilateral triangle";
     } else if (side1 === side2 || side1 === side3 || side2 === side3) {
```

Comment:
remove this else also, just `return`

Reply:
@AlenaBDE did you mean it?

![image](https://user-images.githubusercontent.com/70585151/222342755-5fe608cd-4929-4987-a5f5-2fe557f99ef6.png)

Reply:
if mean return after each `if`, no `else` part

Reply:
@AlenaBDE updated